Socials.yml

```diff
@@ -67,23 +30,63 @@ Google%2B:
   - url.google.com
   - com.google.android.apps.plus
 
-douban:
+Douban:
   - douban.com
 
 Dribbble:
   - dribbble.com
 
-Odnoklassniki:
-  - odnoklassniki.ru
+Facebook:
+  - facebook.com
+  - fb.me
+  - m.facebook.com
+  - l.facebook.com
 
-Viadeo:
-  - viadeo.com
+Fetlife:
+  - fetlife.com
 
 Flickr:
   - flickr.com
 
-WeeWorld:
-  - weeworld.com
+Flixster:
+  - flixster.com
+
+Fotolog:
+  - fotolog.com
+
+Foursquare:
+  - foursquare.com
+
+Friends Reunited:
```

Comment:
Have you tested if removing the surrounding `"` might break Piwik? I'm not 100% sure if the yml parser used in Piwik would parse that correctly.